client/verta/verta/_registry/modelversion.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import print_function
+
+from .._tracking.entity import _ModelDBEntity
+
+
+class RegisteredModelVersion(_ModelDBEntity):
+    def __init__(self, conn, conf, msg):
+        raise NotImplementedError
+
+    def __repr__(self):
+        raise NotImplementedError
+
+    @property
+    def name(self):
+        self._refresh_cache()
+        return self._msg.name
+
+    @classmethod
+    def _generate_default_name(cls):
+        return "ModelVersion {}".format(_utils.generate_default_name())
+
+    @classmethod
+    def _get_proto_by_id(cls, conn, id):
+        raise NotImplementedError
+
+    @classmethod
+    def _get_proto_by_name(cls, conn, name, workspace):
```

Comment:
```suggestion
    def _get_proto_by_name(cls, conn, name, model_id):
```
Similar to how [the signature for `ExperimentRun`](https://github.com/VertaAI/modeldb/blob/cm/registry-skeleton/client/verta/verta/_tracking/experimentrun.py#L170) uses `name`+`expt_id`